src/server/session.ts

```diff
@@ -809,7 +809,7 @@ namespace ts.server {
                 // The rename info should be the same for every project
                 const renameInfo = defaultProject.getLanguageService().getRenameInfo(file, position);
                 if (!renameInfo) {
-                    return undefined;
+                    return emptyArray;

```

Comment:
Are we confident that callers expecting a `protocol.RenameResponseBody` won't hit this case?